diff --git a/lib/utils/http-utils.js b/lib/utils/http-utils.js
index 75fac57f35269e910eb946c85a15f3ef21a40404..4bea73283a6ceee0c2e860ac3c25a6c051896f05 100644
--- a/lib/utils/http-utils.js
+++ b/lib/utils/http-utils.js
@@ -1,4 +1,4 @@
-import fetch from 'cross-fetch';
+import { fetch } from 'cross-fetch';
 import { mergeAbortSignals, TimedAbortSignal, abortable } from './abort-signal-utils.js';
 const DEFAULT_FETCH_TIMEOUT = 60 * 1000 * 3;
 export var HttpMethods;