diff --git a/dist/baseX.js b/dist/baseX.js
index aece06790d04969d4cfe129aafdcf1eb52facd59..63860c5f134a9dd99ea3be047ea332a6fdb98d95 100644
--- a/dist/baseX.js
+++ b/dist/baseX.js
@@ -1,8 +1,9 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.base64url = exports.base58btc = void 0;
-const baseX = require("base-x");
-const BASE58 = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
+import baseX from "base-x";
+const BASE58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 exports.base58btc = baseX(BASE58);
-const BASE64URL = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_';
+const BASE64URL =
+  "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_";
 exports.base64url = baseX(BASE64URL);
diff --git a/dist/esm/baseX.js b/dist/esm/baseX.js
index 094ef4ef1dd1da6a6353ea3b5ef15eaf204f69e4..b26f0992a6d6f69d40ee6991f60472ef918ede22 100644
--- a/dist/esm/baseX.js
+++ b/dist/esm/baseX.js
@@ -1,5 +1,6 @@
-import * as baseX from 'base-x';
-const BASE58 = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
+import baseX from "base-x";
+const BASE58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 export const base58btc = baseX(BASE58);
-const BASE64URL = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_';
+const BASE64URL =
+  "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_";
 export const base64url = baseX(BASE64URL);