diff --git a/dist/baseX.d.ts b/dist/baseX.d.ts
index a8cd49a0cd24872e0b82fa0b55a477b5eb54fafc..8542f67274be18ce30f90e9abd3da71d1620bee2 100644
--- a/dist/baseX.d.ts
+++ b/dist/baseX.d.ts
@@ -1,3 +1,3 @@
-import * as baseX from 'base-x';
+import baseX from "base-x";
 export declare const base58btc: baseX.BaseConverter;
 export declare const base64url: baseX.BaseConverter;
diff --git a/dist/esm/baseX.js b/dist/esm/baseX.js
index 094ef4ef1dd1da6a6353ea3b5ef15eaf204f69e4..b26f0992a6d6f69d40ee6991f60472ef918ede22 100644
--- a/dist/esm/baseX.js
+++ b/dist/esm/baseX.js
@@ -1,5 +1,6 @@
-import * as baseX from 'base-x';
-const BASE58 = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
+import baseX from "base-x";
+const BASE58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 export const base58btc = baseX(BASE58);
-const BASE64URL = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_';
+const BASE64URL =
+  "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_";
 export const base64url = baseX(BASE64URL);