diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 135baace0452dc27edd1e87ca295f63ec69550fb..0000000000000000000000000000000000000000
diff --git a/dist/types/global.d.ts b/dist/types/global.d.ts
index b877eefc00379298bbc165b0481794edbe7509a2..c11eb088e84a9143edd35284ed3ee9441930d4f6 100644
--- a/dist/types/global.d.ts
+++ b/dist/types/global.d.ts
@@ -1,6 +1,6 @@
 import type { MetaMaskInpageProvider } from '@metamask/providers';
 import type { SnapsGlobalObject } from '@metamask/snaps-rpc-methods';
 declare global {
-    const ethereum: MetaMaskInpageProvider;
-    const snap: SnapsGlobalObject;
+    var ethereum: MetaMaskInpageProvider;
+    var snap: SnapsGlobalObject;
 }