diff --git a/build/action-handler.js b/build/action-handler.js
index da2ef3ea0c83e2b0af6cf3b20fe86197e0ebfb17..dfa77234f8daec940d32f4f2169745c71656bd32 100644
--- a/build/action-handler.js
+++ b/build/action-handler.js
@@ -70,7 +70,7 @@ export class CredentialPlugin {
             throw new Error('invalid_argument: presentation.holder must be a DID managed by this agent');
         }
         //FIXME: `args` should allow picking a key or key type
-        const key = identifier.keys.find((k) => k.type === 'Secp256k1' || k.type === 'Ed25519');
+        const key = identifier.keys.find((k) => k.type === 'Secp256k1' || k.type === 'Ed25519' || k.type === 'Secp256r1');
         if (!key)
             throw Error('key_not_found: No signing key for ' + identifier.did);
         let verifiablePresentation;
@@ -103,6 +103,8 @@ export class CredentialPlugin {
             let alg = 'ES256K';
             if (key.type === 'Ed25519') {
                 alg = 'EdDSA';
+            } else if (key.type === 'Secp256r1') {
+                alg = 'ES256';
             }
             const signer = wrapSigner(context, key, alg);
             const jwt = await createVerifiablePresentationJwt(presentation, { did: identifier.did, signer, alg }, { removeOriginalFields, challenge, domain, ...otherOptions });