diff --git a/build/index.js b/build/index.js
index 2d2293decc61f1f3230710296ae68964dded1063..ebddf77d7d5b42d1cd44468f318e955fd57f5ce1 100644
--- a/build/index.js
+++ b/build/index.js
@@ -10,12 +10,12 @@
  *
  * @packageDocumentation
  */
-export { CredentialIssuerLD } from './action-handler.js';
-export * from './types.js';
-export { LdDefaultContexts } from './ld-default-contexts.js';
-export { VeramoLdSignature } from './ld-suites.js';
-export * from './suites/EcdsaSecp256k1RecoverySignature2020.js';
-export * from './suites/Ed25519Signature2018.js';
-export * from './suites/Ed25519Signature2020.js';
-export * from './suites/JsonWebSignature2020.js';
+export { CredentialIssuerLD } from "./action-handler.js";
+export * from "./types.js";
+export { LdDefaultContexts } from "./ld-default-contexts.js";
+export { VeramoLdSignature } from "./ld-suites.js";
+export * from "./suites/EcdsaSecp256k1RecoverySignature2020.js";
+//export * from './suites/Ed25519Signature2018.js';
+export * from "./suites/Ed25519Signature2020.js";
+//export * from './suites/JsonWebSignature2020.js';
 //# sourceMappingURL=index.js.map