diff --git a/node_modules/@veramo/data-store/build/identifier/did-store.d.ts b/node_modules/@veramo/data-store/build/identifier/did-store.d.ts
index a4bdec0..34c8b12 100644
--- a/node_modules/@veramo/data-store/build/identifier/did-store.d.ts
+++ b/node_modules/@veramo/data-store/build/identifier/did-store.d.ts
@@ -12,7 +12,7 @@ export declare class DIDStore extends AbstractDIDStore {
     delete({ did }: {
         did: string;
     }): Promise<boolean>;
-    import(args: IIdentifier): Promise<boolean>;
+    importKey(args: IIdentifier): Promise<boolean>;
     list(args: {
         alias?: string;
         provider?: string;
diff --git a/node_modules/@veramo/data-store/build/identifier/did-store.js b/node_modules/@veramo/data-store/build/identifier/did-store.js
index f3840da..b780c3f 100644
--- a/node_modules/@veramo/data-store/build/identifier/did-store.js
+++ b/node_modules/@veramo/data-store/build/identifier/did-store.js
@@ -98,7 +98,7 @@ class DIDStore extends did_manager_1.AbstractDIDStore {
             return true;
         });
     }
-    import(args) {
+    importKey(args) {
         return __awaiter(this, void 0, void 0, function* () {
             const identifier = new identifier_1.Identifier();
             identifier.did = args.did;
diff --git a/node_modules/@veramo/data-store/build/identifier/key-store.d.ts b/node_modules/@veramo/data-store/build/identifier/key-store.d.ts
index 457ef33..b452dd7 100644
--- a/node_modules/@veramo/data-store/build/identifier/key-store.d.ts
+++ b/node_modules/@veramo/data-store/build/identifier/key-store.d.ts
@@ -10,7 +10,7 @@ export declare class KeyStore extends AbstractKeyStore {
     delete({ kid }: {
         kid: string;
     }): Promise<boolean>;
-    import(args: IKey): Promise<boolean>;
+    importKey(args: IKey): Promise<boolean>;
     list(args?: {}): Promise<ManagedKeyInfo[]>;
 }
 //# sourceMappingURL=key-store.d.ts.map
\ No newline at end of file
diff --git a/node_modules/@veramo/data-store/build/identifier/key-store.js b/node_modules/@veramo/data-store/build/identifier/key-store.js
index 5841494..483fd92 100644
--- a/node_modules/@veramo/data-store/build/identifier/key-store.js
+++ b/node_modules/@veramo/data-store/build/identifier/key-store.js
@@ -40,7 +40,7 @@ class KeyStore extends key_manager_1.AbstractKeyStore {
             return true;
         });
     }
-    import(args) {
+    importKey(args) {
         return __awaiter(this, void 0, void 0, function* () {
             const key = new key_1.Key();
             key.kid = args.kid;
diff --git a/node_modules/@veramo/data-store/build/identifier/private-key-store.d.ts b/node_modules/@veramo/data-store/build/identifier/private-key-store.d.ts
index 5cedbfe..0743830 100644
--- a/node_modules/@veramo/data-store/build/identifier/private-key-store.d.ts
+++ b/node_modules/@veramo/data-store/build/identifier/private-key-store.d.ts
@@ -11,7 +11,7 @@ export declare class PrivateKeyStore extends AbstractPrivateKeyStore {
     delete({ alias }: {
         alias: string;
     }): Promise<boolean>;
-    import(args: ImportablePrivateKey): Promise<ManagedPrivateKey>;
+    importKey(args: ImportablePrivateKey): Promise<ManagedPrivateKey>;
     list(): Promise<Array<ManagedPrivateKey>>;
 }
 //# sourceMappingURL=private-key-store.d.ts.map
\ No newline at end of file
diff --git a/node_modules/@veramo/data-store/build/identifier/private-key-store.js b/node_modules/@veramo/data-store/build/identifier/private-key-store.js
index 13c4f53..f816e99 100644
--- a/node_modules/@veramo/data-store/build/identifier/private-key-store.js
+++ b/node_modules/@veramo/data-store/build/identifier/private-key-store.js
@@ -48,7 +48,7 @@ class PrivateKeyStore extends key_manager_1.AbstractPrivateKeyStore {
             return true;
         });
     }
-    import(args) {
+    importKey(args) {
         return __awaiter(this, void 0, void 0, function* () {
             const key = new private_key_1.PrivateKey();
             key.alias = args.alias || (0, uuid_1.v4)();
