diff --git a/dist/browser-ponyfill.js b/dist/browser-ponyfill.js
index f216aa35a9d32f86df5701c78cf7b039bc2cd47c..a5f764e94a12b2848dd1938b00117a8fe20614f8 100644
--- a/dist/browser-ponyfill.js
+++ b/dist/browser-ponyfill.js
@@ -543,12 +543,22 @@ __self__.fetch.ponyfill = true;
 // Remove "polyfill" property added by whatwg-fetch
 delete __self__.fetch.polyfill;
 // Choose between native implementation (global) or custom implementation (__self__)
-// var ctx = global.fetch ? global : __self__;
-var ctx = __self__; // this line disable service worker support temporarily
-exports = ctx.fetch // To enable: import fetch from 'cross-fetch'
-exports.default = ctx.fetch // For TypeScript consumers without esModuleInterop.
-exports.fetch = ctx.fetch // To enable: import {fetch} from 'cross-fetch'
-exports.Headers = ctx.Headers
-exports.Request = ctx.Request
-exports.Response = ctx.Response
+var ctx = global.fetch ? global : __self__;
+// var ctx = __self__; // this line disable service worker support temporarily
+// exports = ctx.fetch // To enable: import fetch from 'cross-fetch'
+// exports.default = ctx.fetch // For TypeScript consumers without esModuleInterop.
+// exports.fetch = ctx.fetch // To enable: import {fetch} from 'cross-fetch'
+// exports.Headers = ctx.Headers
+// exports.Request = ctx.Request
+// exports.Response = ctx.Response
+
+exports = {
+  ...ctx.fetch,
+  default: ctx.fetch,
+  fetch: ctx.fetch,
+  Headers: ctx.Headers,
+  Request: ctx.Request,
+  Response: ctx.Response,
+};
+
 module.exports = exports